fickling/analysis.py

```diff
@@ -314,26 +313,35 @@ def check_safety(
 
     results = analyzer.analyze(pickled)
     analysis_message = results.to_string(verbosity)
-    stdout.write(analysis_message)
+    # We want nothing printed by DEFAULT but we want users to enable if needed
+    if print_results:
+        stdout.write(analysis_message)
```

Comment:
We should have a separate function for printing results, that can be called by users if they really want to. Like 
```
def print_check_results(results: AnalysisResults, verbosity):
    print(results.to_string(verbosity))
```